### ISSUE TITLE
[Request] Add functionality to BrushFactory to create brush from 2D polygon + height

### ISSUE THREAD

**Issue by LogicalError** [LogicalError/realtime-CSG-for-unity#57]
It would be nice to have the ability to create a brush from a 2D polygon + height from code, using a method in BrushFactory.

**Comment by HandsomeHank** [comment]
BuildR does this.  Maybe you can look at that code to see how to do it?

**Comment by LogicalError** [comment]
RealtimeCSG also does this already, this ticket is to make it possible to do this from code ...

**Comment by HandsomeHank** [comment]
Yeah, BuildR can do it from code. For example, it can connect to a program called mapity and automatically generate a city full of buildings at run time.
